Lib/test/test_source_encoding.py

```diff
@@ -300,5 +295,18 @@ def check_script_output(self, src, expected):
         self.assertEqual(res.out.rstrip(), expected)
 
 
+class ImportTests(unittest.TestCase):
+    def setUp(self):
+        mod = importlib.import_module('test.encoded_modules')
+        self.test_strings = mod.test_strings
+        self.test_path = mod.__path__
```

Comment:
`test_path` is unused, and `test_strings` is only used in this one test, which can also be added to `MiscSourceEncodingTest` in place of `test_pep263`.  See suggestion above.